src/test/java/seedu/address/storage/StorageManagerTest.java

```diff
@@ -54,7 +54,7 @@ public void addressBookReadSave() throws Exception {
          * {@link JsonAddressBookStorage} class.
          * More extensive testing of UserPref saving/reading is done in {@link JsonAddressBookStorageTest} class.
          */
-        InternBook original = getTypicalAddressBook();
+        InternBook original = getTypicalInternBook();
         storageManager.saveAddressBook(original);
```

Comment:
in the storagemanger class perhaps you could change the method saveAddressBook to better fit our context